.github/workflows/build_and_test.yml

```diff
@@ -39,3 +39,33 @@ jobs:
 
       - name: Run unit tests
         run: yarn test-unit
+
+  build:
+    runs-on: ${{ matrix.os }}
+    name: Build on ${{ matrix.os }}
+    needs: lint-test
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest]
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: '.nvmrc'
+          registry-url: 'https://registry.npmjs.org'
+          cache: 'yarn'
+
+      - name: Setup Yarn
+        run: |
+          npm uninstall -g yarn
+          npm i -g yarn@1.22.10
+          yarn install --frozen-lockfile
+
+      - name: Build
+        run: yarn build
```

Comment:
Can we skip this for PRs with no that only have docs changes, or otherwise scope it to only necessary PRs?

Reply:
Created #828